config/locales/en.yml

```diff
@@ -43,6 +43,7 @@ en:
                 other: 'Missing required oauth fields in manifest: %{missing_keys}'
             missing_source: Could not find app.js
             style_in_template: "<style> tag in %{template}. Use an app.css file instead."
+            no_code_for_ifo_notemplate: Javascripts, stylesheets and templates are not allowed when an iframe URI or noTemplate is specified
```

Comment:
Would it be possible to get a screenshot for translators? What is noTemplate? Written together? Should it be localized or not? Thanks!